### ISSUE TITLE
Non-flag additional arguments?

### ISSUE THREAD

**Issue by dbushong** [relistan/troll-opt#1]
Is there a way to specify that additional non-flag arguments are allowed?  E.g.:

```
./my-script file1 file2 file3
```

(or, with boolean flag -c specified):

```
./my-script -c file1 file2 file3
```

In Trollop it looks like you can just read ARGV after you're done, but with troll-opt when I add arguments to the end of my command line I get: 

```
./my-script file1 file2 file3
ERROR: Error: Unknown argument or a value supplied for flag: file1
```

**Comment by relistan** [comment]
You are right, it doesn't behave like Trollop in this way.  But it should.  I'll see if I can manage to have a crack at it this weekend.  Or I'd be happy to accept a pull request!  Cheers

**Comment by relistan** [comment]
I fixed this and released as 0.7.0.  The readme now covers how to get to the additional args.
